great_expectations/datasource/fluent/fluent_base_model.py

```diff
@@ -69,7 +66,265 @@ class Config:
         extra = pydantic.Extra.forbid
 
     @classmethod
-    def parse_yaml(cls: Type[_Self], f: Union[pathlib.Path, str]) -> _Self:
+    def parse_yaml(cls: Type[Self], f: Union[pathlib.Path, str]) -> Self:
+        loaded = yaml.load(f)
+        logger.debug(f"loaded from yaml ->\n{pf(loaded, depth=3)}\n")
+        # noinspection PyArgumentList
+        config = cls(**loaded)
+        return config
+
+    @overload
```

Comment:
Non-blocking, but I don't love the duplication here. I don't like using multiple inheritance, but maybe the best option here is for `GenericBaseModel` to inherit from `FluentBaseModel` and `pydantic.GenericModel`? Looks like `pydantic.GenericModel inherits from `pydantic.BaseModel`, so I assume it'd work just fine.

Reply:
if that works, that's great! added in [ef4651e](https://github.com/great-expectations/great_expectations/pull/9785/commits/ef4651e80793253928d7cf9e8755cd9b2c74debc)